_rules/iframe-not-focusable-has-no-interactive-content-akn7bn.md

```diff
@@ -28,13 +28,17 @@ acknowledgments:
 
 ## Applicability
 
-This rule applies to any `iframe` element that [contains](#akn7bn:contain) at least one element for which all the following are true:
+This rule applies to any `iframe` element that is not [inert](#akn7bn:inert) and that [contains](#akn7bn:contain) at least one element for which all the following are true:
 
 - the element is [visible][]; and
 - the element is part of the [sequential focus navigation order][] of the `iframe`'s [document][].
 
 An element is <dfn id="akn7bn:contain">contained</dfn> in a [nested browsing context][] if its [owner document][] is the [container document][] of the [nested browsing context][].
 
+An `iframe` element is <dfn id="akn7bn:inert">inert</dfn> if:
+- it has an `inert` [attribute value][] of true; or
```

Comment:
The attribute value doesn't matter. It just has to be present. Plus this should apply to ancestors too. Inert disables the entire subtree.

Reply:
> The attribute value doesn't matter. It just has to be present. 

As per [our definition of attribute value](https://github.com/act-rules/act-rules.github.io/blob/develop/pages/glossary/attribute-value.md), boolean attribute (like `inert`) have an "attribute value" of `true` if the attribute is present.